test.mk

```diff
@@ -161,50 +161,255 @@ spec:
           serviceName: {{NAME}}
           servicePort: 80
 endef
+define TEST_SERVICE_SPEC_INGRESS_CLOUDEP
+$(call TEST_INGRESS)
+--- 
+$(call TEST_SERVICE)
+  targetIngress:
+    name: {{INGRESS_NAME}}
+    namespace: default
+    jwtServices:
+    - {{NAME}}
+---
+$(call TEST_APP)
+endef
+define TEST_SERVICE_SPEC_CONFIGMAP_SPEC
+apiVersion: v1
+kind: ConfigMap 
+metadata: 
+  name: swagger 
+data: 
+  spec: |
+    swagger: "2.0"
+    info:
+      description: "wildcard config for any HTTP service."
+      title: "ConfigMap Template: {{NAME}}"
+      version: "1.0.0"
+    basePath: "/"
+    consumes:
+    - "application/json"
+    produces:
+    - "application/json"
+    schemes:
+    - "http"
+    - "https"
+    paths:
+      "/**":
+        get:
+          operationId: Get
+          responses:
+            '200':
+              description: Get
+            default:
+              description: Error
+        delete:
+          operationId: Delete
+          responses:
+            '204':
+              description: Delete
+            default:
+              description: Error
+        patch:
+          operationId: Patch
+          responses:
+            '200':
+              description: Patch
+            default:
+              description: Error
+        post:
+          operationId: Post
+          responses:
+            '200':
+              description: Post
+            default:
+              description: Error
+        put:
+          operationId: Put
+          responses:
+            '200':
+              description: Put
+            default:
+              description: Error
+    security:
```

Comment:
wrap the `security` and `securityDefinitions` blocks in a template conditional to fill in JWTAudience from the template variables.